etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -55,6 +55,11 @@ spec:
         - name: POSTGRES_SSL
           value: {{ .Values.global.postgresql.ssl | quote }}
         {{- end }}
+        - name: POSTGRES_PASSWORD
+          valueFrom:
+            secretKeyRef:
```

Comment:
Very Minor: Could maybe do a `envFrom: - secretRef:` here so if additional secrets are ever added, don't need to add additional wiring.